### PR TITLE
[CBRD-24477] Add PL/CSQL development phase-0 testcases

### PR DESCRIPTION
**Pull request by hyunikn**
http://jira.cubrid.org/browse/CBRD-24477

- Testcases which amounts to the development phase-0 and the Test Specification Document 
- All the cases are OK for the current commit (5089f04bc) of the CUBRID feature/plcsql branch

### CODE REVIEW

File: sql/_05_plcsql/_01_testspec/_01_basic_structure/_01_create_procedure/_01_wo_or_replace/cases/01_01_01-01_normal_basic.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+--+ server-message on
```
Comment (hiclass) [inline sql/_05_plcsql/_01_testspec/_01_basic_structure/_01_create_procedure/_01_wo_or_replace/cases/01_01_01-01_normal_basic.sql:1]: --+ server-output on
--+ server-message on

which one, would be used?

Comment (hgryoo) [reply sql/_05_plcsql/_01_testspec/_01_basic_structure/_01_create_procedure/_01_wo_or_replace/cases/01_01_01-01_normal_basic.sql:1]: Only the server-message will be used. Please note that the hint is going to be changed from 'server-output' to 'server-message'  in the PR (https://github.com/CUBRID/cubrid-testtools/pull/662)
```
--+ server-message on
```

**Review by beyondykk9 — APPROVED** [review]

File: sql/_05_plcsql/_01_testspec/_02_declaration/_07_local_proc/_05_common/cases/02_07_05-03_normal_return_with_value.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
+--+ server-message on
+
+-- error: local procedure returns with a value
+
```
Comment (kwonhoil) [inline sql/_05_plcsql/_01_testspec/_02_declaration/_07_local_proc/_05_common/cases/02_07_05-03_normal_return_with_value.sql]: The file name must be changed as shown below.

02_07_05-03_error_return_with_value.sql

File: sql/_05_plcsql/_01_testspec/_02_declaration/_07_local_proc/_05_common/answers/02_07_05-03_normal_return_with_value.answer [hunk 3]
```diff
@@ -0,0 +1,13 @@
+===================================================
+Error:-1359
```
Comment (kwonhoil) [inline sql/_05_plcsql/_01_testspec/_02_declaration/_07_local_proc/_05_common/answers/02_07_05-03_normal_return_with_value.answer]: The file name must be changed as shown below.

02_07_05-03_error_return_with_value.answer

File: sql/_05_plcsql/_01_testspec/_03_statement/_07_exit/_01_with_label/cases/03_07_01-01_normal_basic.sql [hunk 4]
```diff
@@ -0,0 +1,29 @@
+--+ server-message on
+
+-- normal: basic usage of exit statements with labels
+
+
+create or replace procedure t(i int) as
+begin
+    <<outer>>
```
Comment (kwonhoil) [inline sql/_05_plcsql/_01_testspec/_03_statement/_07_exit/_01_with_label/cases/03_07_01-01_normal_basic.sql:8]: “Outer” and “inner” are reserved words. Can they be used as label names?

**Review by kwonhoil — APPROVED** [review]

**Review by hiclass — APPROVED** [review]

File: sql/_05_plcsql/_01_testspec/_02_declaration/_07_local_proc/_05_common/cases/02_07_05-03_normal_return_with_value.sql (hunk 2 above)
Comment (hyunikn) [reply sql/_05_plcsql/_01_testspec/_02_declaration/_07_local_proc/_05_common/cases/02_07_05-03_normal_return_with_value.sql]: Yes. Thank you

File: sql/_05_plcsql/_01_testspec/_03_statement/_07_exit/_01_with_label/cases/03_07_01-01_normal_basic.sql (hunk 4 above)
Comment (hyunikn) [reply sql/_05_plcsql/_01_testspec/_03_statement/_07_exit/_01_with_label/cases/03_07_01-01_normal_basic.sql:8]: Some reserved words of CUBRID are not reserved words of PL/CSQL, while they are reserved words if they appear in a Static/Dynamic SQL.
PL/CSQL reserved words are listed in http://192.168.2.208/ko/pl.html#id3, although the list needs some updates.

File: sql/_05_plcsql/_01_testspec/_02_declaration/_07_local_proc/_05_common/answers/02_07_05-03_normal_return_with_value.answer (hunk 3 above)
Comment (hyunikn) [reply sql/_05_plcsql/_01_testspec/_02_declaration/_07_local_proc/_05_common/answers/02_07_05-03_normal_return_with_value.answer]: Yes, Thank you

**Review by hgryoo — APPROVED** [review]

**Pull request merged**